src/ukify/ukify.py

```diff
@@ -1686,7 +1738,9 @@ def __call__(
         '--pcr-private-key',
         dest='pcr_private_keys',
         action='append',
-        help='private part of the keypair or engine-specific designation for signing PCR signatures',
+        help=(
+            'private part of the keypair or engine/provider specific designation for signing PCR signatures'
+        ),
```

Comment:
Nit: I think it would be more in line with the rest to not have this in parentheses and put a `# noqa: E501` at the end.